_test/check-exercises.escript

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env escript
+
+main( [] ) -> 
+	Examples = filelib:wildcard( "*/example.erl" ),
+	Modules = [{X, compile(X)} || X <- Examples],
+	[compile_tests(X) || X <- Modules],
+	Results = [run_tests(X) || X <- Modules],
+	erlang:halt( erlang:length([X || X <- Results, X =:= ok]) );
```

Comment:
This is returning the number of passed tests, http://www.erlang.org/doc/man/erlang.html#halt-1
It should be checking for failures i.e.
```erlang
  erlang:halt( erlang:length([X || X <- Results, X =/= ok]) );
```